mlkem/kem.c

```diff
@@ -178,6 +183,8 @@ static int mlk_check_pct(uint8_t const pk[MLKEM_INDCCA_PUBLICKEYBYTES],
 }
 #endif /* MLKEM_KEYGEN_PCT */
 
+/* Reference: `crypto_kem_keypair_derand()` in the reference implementation
+ *            - We include PCT which is not present in the reference code. */
```

Comment:
we disable PCT by default, please be specific about that here.

Reply:
Right